### DISCUSSION TITLE
lab 02.04 Task 1. Make the heaps

### DISCUSSION THREAD

**Discussion by carlosdac55** [rramosp/ai4eng.v1#27] [02 - PYTHON]
Buenas tardes profesor, primero quiero darle la gracias por toda la información valiosa que nos comparte en este curso, con respecto al la tarea Task 1. Make the heaps, tengo un problema, pues al correr el código, obtengo un resultado, sin embargo en la autocorrección solo tengo que mi resultado es cero (la única diferencia con respecto al resultado esperado es el orden de las letras) , no obtengo ningún mensaje que me indique en que ha fallado, el código se ejecuta bien y por lo tanto le solicito ayuda para entender en que esta fallando mi código.

def make_heaps1(c, n_heaps=3):
    assert n_heaps%2==1, "must have an odd number of heaps"
    assert len(c)%n_heaps==0, "the length of the deck must be a multiple of the number of heaps"
    split_c=np.array_split(c,n_heaps)
    split_per=np.random.permutation(split_c)
    h=[]
    for i in split_per:
      y=list(i)
      h.append(y)
    return h

![resultado](https://user-images.githubusercontent.com/85840609/122610187-d3851580-d044-11eb-8eb4-3d39fe135bc8.JPG)

Agradezco mucho su colaboración.

**Comment by carlosdac55** [comment]
Solucionado pues me di cuenta que el error era la manera de barajar las cartas, corregí el código y ya tengo el resultado deseado.

**Reply by rramosp** [reply]
perfecto Carlos! gracias!!
